app/styles/base.less

```diff
@@ -347,7 +347,7 @@ thead {
   width: 22px;
   height: 22px;
   display: inline-block;
-  background: url(img/log.png) no-repeat;
+  background: url(../img/log.png) no-repeat;
```

Comment:
Looks like it's not working as I'd hope to: https://github.com/HenrikJoreteg/hjs-webpack/issues/119 will need to fix this before we can merge